recursion/recursivemultiplier.js

```diff
@@ -0,0 +1,13 @@
+const recursiveMultiplier = (arr, num) => {
+    if(arr.length === 0) {
```

Comment:
you forgot to check if this is indeed an array.

Reply:
i had assumed it to be a non null array